### PR TITLE
EVG-6497 add logging for end task

### PR DESCRIPTION
**Pull request by john-m-liu**

### CODE REVIEW

File: model/task_lifecycle.go [hunk 1]
```diff
@@ -398,6 +398,7 @@ func doStepback(t *task.Task) error {
 // MarkEnd updates the task as being finished, performs a stepback if necessary, and updates the build status
 func MarkEnd(t *task.Task, caller string, finishTime time.Time, detail *apimodels.TaskEndDetail,
 	deactivatePrevious bool, updates *StatusChanges) error {
+	const slowThreshold = 1 * time.Second
```
Comment (tychoish) [inline model/task_lifecycle.go]: s/1 \*//

File: model/task_lifecycle.go [hunk 2]
```diff
@@ -418,7 +419,15 @@ func MarkEnd(t *task.Task, caller string, finishTime time.Time, detail *apimodel
 			"activated_by": t.ActivatedBy,
 		})
 	}
+	now := time.Now()
```
Comment (tychoish) [inline model/task_lifecycle.go]: now -> startPhaseAt

**Comment by tychoish** [comment]
Also make the tests pass

File: model/task_lifecycle.go (hunk 1 above)
Comment (john-m-liu) [reply model/task_lifecycle.go]: I have no idea what this means

File: model/task_lifecycle.go (hunk 2 above)
Comment (john-m-liu) [reply model/task_lifecycle.go]: renamed

**Pull request merged**